kubernetes/ama-logs.yaml

```diff
@@ -984,6 +984,33 @@ spec:
           - name: ndots
             value: "3"
      containers:
+      # Uncomment below lines for MSI Auth Mode  testing
+      #  - name: addon-token-adapter-win
+      #    command:
+      #     - addon-token-adapter-win
+      #    args:
+      #     - --secret-namespace=kube-system
+      #     - --secret-name=aad-msi-auth-token
+      #     - --token-server-listening-port=7777
+      #     - --health-server-listening-port=9999
+      #    image: "mcr.microsoft.com/aks/hcp/addon-token-adapter:20230120winbeta"
```

Comment:
If this is not latest windows token adapter image, please update it.

Reply:
This is the latest